### ISSUE TITLE
NX-OS: show vlan. Can not parse vlan with name that contains underscore "_"

### ISSUE THREAD

**Issue by urskog84** [CiscoTestAutomation/genieparser#17]
`genie parse "show vlan" --testbed-file dc.yml --output /dc_vlan`

the file <devicname>_console.txt includes vlan with underscore in the name
```
27   0027_192.168.1.0/24              active    Po1, Po3, Po4, Po5, Po7, Po9
                                                Po11, Po12, Po13, Po15, Eth1/1
                                                Eth1/2, Eth1/3, Eth1/4, Eth1/6
                                                Eth1/7, Eth1/9, Eth1/10, Eth1/11
                                                Eth1/15, Eth1/41, Eth1/43
                                                Eth1/44, Eth1/45, Eth1/46
                                                Eth2/1, Eth2/2, Eth122/1/21
                                                Eth122/1/22
28   0028_192.168.5.0/24              active    Po1, Po3, Po4, Po5, Po7, Po9
                                                Po11, Po12, Po13, Po15, Eth1/1
                                                Eth1/2, Eth1/3, Eth1/4, Eth1/6
                                                Eth1/7, Eth1/9, Eth1/10, Eth1/11
                                                Eth1/15, Eth1/41, Eth1/43
                                                Eth1/44, Eth1/45, Eth1/46
                                                Eth2/1, Eth2/2, Eth122/1/21
                                                Eth122/1/22
```

but i don't get any vlan 27 or 28 int the parsted file. <devicname>_parsed.txt

if I remove the _ (undersore) form vlan name it ends upp in the <devicname>_parsed.txt file

**Comment by tahigash** [comment]
Hi @urskog84 Thanks for reporting this. We will look into this and fix.

**Comment by urskog84** [comment]
Thanks for the quick response.
@tahigash have you bin able to reproduce the "bug"? Let me now if you need more information from me,
ex. os version / switch model.. etc

**Comment by tahigash** [comment]
@urskog84 yes. we saw the issue with given snipped output. we are now working on the fix and beta package with the fix will be provided soon.

**Comment by tahigash** [comment]
@urskog84 We've released beta package 19.4.1b1 with the fix. please check if it works.

pip install genie.libs.parser --upgrade --pre

Regards,

**Comment by urskog84** [comment]
I have update to 19.4.1b1.
but now I get   `Could not parse 'show vlan' `

```
root@920ae6ff8b6d:/pyats# pip list genie.libs.parser | grep genie.libs.parser
genie.libs.parser            19.4.1b1
```

+=================================================================+
| Genie Parse Summary for N9KDH2-2                                             |
+=================================================================+
|  Connected to N9KDH2-2                                                       |
|  -  Log: /pyats/git/genie/dc_vlan/connection_N9KDH2-2.txt                    |
|------------------------------------------------------------------------------|
|  Could not parse 'show vlan'                                                 |
|  -  Exception:      /pyats/git/genie/dc_vlan/N9KDH2-2_show-vlan_exception.txt |
|  -  Device Console: /pyats/git/genie/dc_vlan/N9KDH2-2_show-vlan_console.txt  |
|------------------------------------------------------------------------------|

**Comment by urskog84** [comment]
Hi. i have sent you the requested files via Email!

**Comment by tahigash** [comment]
Hi @urskog84 We received files via email and will look into those.

**Comment by tahigash** [comment]
@urskog84 we fixed the issue and released beta 19.4.1b2. Could you please get the beta by below command and check?

pip install genie.libs.parser --upgrade --pre

Regards,

**Comment by urskog84** [comment]
@tahigash Big thakns. Version 19.4.1b2 works like a charm
